### ISSUE TITLE
[BUG] Ridicuously high Meerkat Spawnrate

### ISSUE THREAD

**Issue by itsdinkd** [How-Bout-No/Outvoted#68]
Latest version of mod
1.17.1mc
fabric

![image](https://user-images.githubusercontent.com/35115304/129823024-efdb5c15-997e-435f-9395-847abbd74061.png)

![image](https://user-images.githubusercontent.com/35115304/129823058-43f7346b-b166-420d-abd0-0eb37399549a.png)

341 meerkats spawned for just me. way past mobcap per player (only 2 people including myself on server, other person is nowhere near me in the desert). 

341 meerkats... 

Please fix spawn rate.

**Comment by How-Bout-No** [comment]
Yikes that's obviously not right. What is the spawn rate (or I guess burrow gen rate) you have in config? Is it the default?

**Comment by itsdinkd** [comment]
Burrow rate was at 50 which had incredibly high spawns so I changed the
burrow rate to 40 and thats when I took the screenshot and reported this.
Both values have incredibly high spawns

**Comment by Raavea** [comment]
Oh I also have this issue!

I don't _entirely_ mind how many there are - my issue is that they're so **LOUD**.

Does anyone have a spawn setting that definitely fixes the overpop? I've not changed it at all yet, everything is at the defaults (well, I'm running All of Fabric 4 so there's a small chance that modpack alters the defaults to its own set of defaults, idk).

(Also, is it possible to alter the volume of their noise in the settings? I took a peep in the .jar but idk what I'm doing.. All I know is even if I have friendly animals so quiet I can barely hear my chickens, the hundreds of meerkats outside my base are still far too loud for comfort.)

**Comment by How-Bout-No** [comment]
> Oh I also have this issue!
> 
> I don't _entirely_ mind how many there are - my issue is that they're so **LOUD**.

I adjusted the volume of all the mobs in the latest alpha builds as most of the mobs were very loud (in my testing I obv only test the mod mobs so I didn't catch any volume issues for a while).
Other than volume, I believe I may have worked out a way for the entity cap issues but it will need some tweaking since if I align the Meerkat spawn cap to the Creature cap then rabbits will immediately use up all the available slots and Meerkats would never spawn. Will need to see if the method I'm using is actually a good idea or not.

**Comment by Raavea** [comment]
Ah I totally understand - when I was making music for my BA, mine were always way quieter than everyone elses because I worked against my own usual volume levels. 

For the time being I've just switched them from the 40:1 to 80:1 in the settings and it seems pretty good at that setting; however, I've not travelled to any new deserts lately.

Like I said, I don't really mind there being lots of them (except the noise startles me) mainly because they seem to be keeping my area spider free - and spiders are gross. :)
Thanks for the swift update <3
Also thanks for making the ocean TERRIFYING, I both hate and love it. Lovely mod.
